src/Schema/Attribute/AttributeDef.php

```diff
@@ -6,27 +6,49 @@
 
 class AttributeDef extends Item implements AttributeItem
 {
+    /**
+     * @var static|null
+     */
     protected $fixed;
 
+    /**
+     * @var static|null
+     */
     protected $default;
 
+    /**
+     * @return static|null
+     */
     public function getFixed()
     {
         return $this->fixed;
     }
 
+    /**
+     * @param static $fixed
+     *
+     * @return $this
+     */
     public function setFixed($fixed)
     {
         $this->fixed = $fixed;
 
         return $this;
     }
 
+    /**
+     * @return static|null
+     */
     public function getDefault()
     {
         return $this->default;
     }
 
+    /**
+     * @param static $default
```

Comment:
here is `string` instead of `static|null`

Reply:
as above.